feat(kms): use kms for also libplanet side

moreal opened the pull request:
This pull request resolves #47

longfin commented on bridge/src/kms-ncg-signer.ts:
```diff
@@ -0,0 +1,102 @@
+      const sequence = new Sequence();
+      // @ts-ignore
+      sequence.valueBlock.value.push(new Integer({ isHexOnly: true, valueHex: r }));
+      console.log("n", nBigInt)
+      console.log('r', bufToBigint(r));
+      console.log('s', sBigInt);
+      console.log('otherS', otherSBigInt);
+      console.log('use otherS', sBigInt > otherSBigInt);
```
```suggestion
```

longfin commented on bridge/src/kms-ncg-signer.ts:
```diff
@@ -0,0 +1,102 @@
+      }
+
+      const sequence_buffer = toBuffer(sequence.toBER(false));
+      console.log('original sign: ', asn1Signature.toString('hex'));
+      console.log('sign: ', sequence_buffer.toString('hex'));
```
```suggestion
```

riemannulus commented on bridge/test/aws-dependent/sign.aws.spec.ts:
```diff
@@ -0,0 +1,60 @@
+
+            const sign = await signer.sign(digest);
+
+            const publicKey = await signer.getPublicKey();
+            console.log(publicKey.toString('base64'));
```
We need to verify the sign here.

moreal replied:
In my thought, this test is useless because the *bridge* application is using `KMSNCGSigner` which implemented by this projecct without `KmsSigner`  implementation. How do you think about this?

longfin reviewed (changes requested):
It seems that headless is failing to verify the signature from AWS KMS yet. 😢

moreal commented:
> 
> 
> It seems that headless is failing to verify the signature from AWS KMS yet. 😢

I don't know your situation fully but I also got situation like you while testing (i.e. running `yarn test`).
I cleaned up code correctly and I believed the mock values are valid. But they were wrong so I updated the values in 9b08973. Could you check it again?

P.S. I'm running endless-loop `yarn test` to try catching edge case error.
